src/lib/components/billing/alerts/selectProjectCloud.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+    import { type Models } from '@appwrite.io/console';
+    import { Alert, Button, Table } from '@appwrite.io/pink-svelte';
+    import { Modal } from '$lib/components';
+    import { onMount } from 'svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+
+    export let showSelectProject: boolean;
+    export let projects: Array<Models.Project> = [];
+    export let selectedProjects: Array<string> = [];
+
+    let error: string | null = null;
+
+    onMount(() => {
+        sdk.forConsole.projects
+            .list()
+            .then((response) => {
+                projects = response.projects || [];
+            })
+            .catch((error) => {
+                console.error('Failed to fetch projects:', error);
+            });
```

Comment:
why don't we use the projects if we have an export named - `export let projects: Array<Models.Project> = [];`. Would be nice to save one request to API.

Reply:
couldn't use it when adding alert from
https://github.com/appwrite/console/blob/b57d0fe0a65bd4c6617c835e65cc624fc1a6c295/src/lib/stores/billing.ts#L295-L296

Reply:
in that case, lets check if we can get projects via `page.data?.projects`, if not available, feel free to make an API request. Also if the exported `projects` variable is not passed/used, remove it.

Reply:
we cannot get from Page.data.projects, as the headeralert is added from layout and we don't have the list of project on layout. removed the export.